tests/unit/test_utilities.py

```diff
@@ -358,3 +361,53 @@ def test_make_array_two_dimensional(self):
 
         self.assertEqual(len(output_array.shape), 2)
         np.testing.assert_array_equal(output_array, expected_output)
+
+
+class TestTransposeIfXdimLessThanYdim(TestCase):
+
+    def test_wider_than_tall(self):
+        # Test case where x dim <  y dim and should transpose
```

Comment:
Nit: If you are putting a comment at the top of a function or a method, you should use a documentation string, not just an in-line comment.

```
def test_wider_than_tall(self):
    """Test case where x dim < y dim and should transpose."""
```

The same is generally true throughout this new test class.